src/mixture_laws.jl

```diff
@@ -76,6 +134,29 @@ function g03_invum(x::Real, Rv::Real)
     )
 end
 
+
+function g03lmc_invum(x::Real, Rv::Real)
+    if !(0.3 <= x <= 10.0)
+        error("out of bounds of G03_LMCAve, support is over $(bounds(G03_LMCAve)) angstrom")
```

Comment:
Recommend making this a `DomainError` for consistency with `g03_invum`
```suggestion
throw(DomainError(x, "out of bounds of G03_LMCAve, support is over $(bounds(G03_LMCAve)) angstrom"))
```

Reply:
Yes, the same as before. Fine for me.